metrics/metrics.go

```diff
@@ -38,6 +38,38 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		}, []string{"result", "strategy", "namespace"})
 
+	PodsEvictedSuccess = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      DeschedulerSubsystem,
+			Name:           "pods_evicted_success",
+			Help:           "Number of pods successfully evicted per strategy",
+			StabilityLevel: metrics.ALPHA,
+		}, []string{"strategy"})
+
+	PodsEvictedFailed = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      DeschedulerSubsystem,
+			Name:           "pods_evicted_failed",
+			Help:           "Number of pods failed to evict per strategy",
+			StabilityLevel: metrics.ALPHA,
+		}, []string{"strategy"})
+
+	TotalPodsEvicted = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      DeschedulerSubsystem,
+			Name:           "total_pods_evicted",
+			Help:           "Total pods evicted so far",
+			StabilityLevel: metrics.ALPHA,
+		}, []string{"result"})
+
+	TotalPodsSkipped = metrics.NewCounterVec(
```

Comment:
is there anywhere you're updating `TotalPodsSkipped`?

Reply:
I was not able to figure out as where I can update `TotalPodsSkipped` if it could be below this [line](https://github.com/kubernetes-sigs/descheduler/blob/8b5bb6a12d9d13b7a3906368137fc37970599cb3/pkg/descheduler/evictions/evictions.go#L122) and this [line](https://github.com/kubernetes-sigs/descheduler/blob/8b5bb6a12d9d13b7a3906368137fc37970599cb3/pkg/descheduler/evictions/evictions.go#L127). If you  could suggest me anything here, will be helpful.

Reply:
Updated.